projects/1.19/assets/lootr/lootr/lang/zh_cn.json

```
@@ -0,0 +1,52 @@
+{
+  "lootr.message.should_sneak": "破坏箱子将会永久删去其中的战利品！如果你真的想破坏它，请在潜行时破坏。",
+  "lootr.message.should_sneak2": "由于箱内战利品是%s，请只在必要时破坏战利品箱。",
+  "lootr.message.should_sneak3": "各玩家相互独立的",
+  "lootr.message.cart_should_sneak": "破坏矿车将会永久删去其中的战利品！如果你真的想破坏它，请在潜行时破坏。",
+  "lootr.message.cart_should_sneak3": "各玩家相互独立的",
+  "lootr.message.cannot_break": "战利品容器和矿车不能被破坏。",
+  "lootr.message.cannot_break_sneak": "战利品容器和矿车只能在创造模式潜行时被破坏。",
+  "lootr.message.decayed": "衰败的容器在你的接触后碎了！",
+  "lootr.message.decay_in": "这个容器将在 %s 秒内完全衰败。",
+  "lootr.message.decay_start": "容器在你的接触后开始碎裂！它将在%s秒内完全衰败。",
+  "lootr.message.refreshed": "容器中的物品在你接触后被刷新！",
+  "lootr.message.refresh_in": "这个容器将在 %s 秒内刷新里面的物品。",
+  "lootr.message.refresh_start": "容器将在 %s 秒后用新的物品进行刷新！",
+  "lootr.message.invalid_table": "这个容器的战利品表 [%s] 为空，战利品无法生成。检查你的 `latest.log` 中是否有报错。",
+  "block.lootr.lootr_chest": "战利品箱子",
+  "block.lootr.lootr_trapped_chest": "战利品箱子",
+  "block.lootr.lootr_barrel": "战利品木桶",
+  "block.lootr.lootr_inventory": "战利品箱子",
+  "block.lootr.lootr_shulker": "战利品潜影贝",
+  "block.lootr.trophy": "百次奖杯",
+  "entity.lootr.lootr_minecart": "运输矿车",
+  "lootr.commands.usage": "/lootr barrel | barrel <loot-table> | chest | chest <loot-table> | cart | cart <loot-table> | shulker | shulker <loot-table> | convert | custom <id> | refresh | decay",
+  "lootr.commands.create": "已使用战利品表 %3$s 于 %2$s 处创建出了 Lootr 的%1$s。",
+  "lootr.commands.summon": "已使用战利品表 %3$s 于 %2$s 处生成出了 Lootr 矿车的%1$s。",
+  "lootr.commands.blockpos": "%s，%s，%s",
+  "stat.lootr.looted_stat": "箱子已搜刮",
+  "lootr.advancements.100loot.title": "百次",
+  "lootr.advancements.100loot.description": "打开 100 次实例化的、各玩家相互独立的战利品容器，获取属于你的战利品！",
+  "lootr.advancements.50loot.title": "五十桨战船",
+  "lootr.advancements.50loot.description": "打开 50 次实例化的、各玩家相互独立的战利品容器，获取属于你的战利品！",
+  "lootr.advancements.25loot.title": "再来！",
+  "lootr.advancements.25loot.description": "打开 25 次实例化的、各玩家相互独立的战利品容器，获取属于你的战利品！",
+  "lootr.advancements.10loot.title": "到处都是",
+  "lootr.advancements.10loot.description": "打开 10 次实例化的、各玩家相互独立的战利品容器，获取属于你的战利品！",
+  "lootr.advancements.1chest.title": "“X”标记着藏宝点",
+  "lootr.advancements.1chest.description": "首次打开实例化的、各玩家相互独立的战利品箱子，获取属于你的战利品！",
+  "lootr.advancements.1barrel.title": "木桶里的战利品",
+  "lootr.advancements.1barrel.description": "首次打开实例化的、各玩家相互独立的战利品木桶，获取属于你的战利品！",
+  "lootr.advancements.1cart.title": "隧道里的宝藏",
+  "lootr.advancements.1cart.description": "首次打开实例化的、各玩家相互独立的战利品矿车，获取属于你的战利品！",
+  "lootr.advancements.1shulker.title": "奖励盒",
+  "lootr.advancements.1shulker.description": "首次打开实例化的、各玩家相互独立的战利品潜影贝，获取属于你的战利品！",
+  "lootr.advancements.social.title": "战利品社会化",
+  "lootr.advancements.social.description": "在世界中首次打开实例化的、各玩家相互独立的战利品容器！",
+  "lootr.advancements.root.title": "老套搜刮已成过去式",
+  "lootr.advancements.root.description": "为你的世界带来实例化的、各玩家相互独立的战利品。",
+  "lootr.advancements.1337loot.title": "/\\/\\4$73R 10073R",
+  "lootr.advancements.1337loot.description": "一共打开了1337个战利品容器! Wow!",

```

```suggestion
  "lootr.advancements.1337loot.description": "一共打开了1337个战利品容器！哇哦！",
```
其余文件也需统一（Wow那个保留原文应该也没关系吧（））
推荐再次查找一下还有没有半角标点